apps/o2-blocks/src/github-issue-template/editor.tsx

```diff
@@ -0,0 +1,277 @@
+		/>
+	</SVG>
+);
+
+interface Label {
+	name: string;
+	color: string;
+}
+/**
+ * Maps repositories to labels or `null` if repository can't be found.
+ */
+type RepoLabels = Map< string, Label[] | null >;
+
+type ShellAs = keyof JSX.IntrinsicElements | ReactT.JSXElementConstructor< any >;
+
+type ShellProps< T extends ShellAs > = Readonly< {
+	as?: T;
+	className?: string;
+	title: ReactT.ReactNode;
+	subtitle: ReactT.ReactNode;
+	body?: ReactT.ReactNode;
+	labels?: Label[];
+} > &
+	ReactT.ComponentProps< T >;
+
+const Shell = < T extends ShellAs = 'div' >( {
+	as: El = 'div',
+	className,
+	title,
+	subtitle,
+	body,
+	labels = [],
+	...elementProps
+}: ShellProps< T > ) => {
+	return (
+		<El { ...elementProps } className={ classNames( 'wp-block-a8c-github-template', className ) }>
+			<div className="wp-block-a8c-github-template__icon" />
+			<div className="wp-block-a8c-github-template__title">{ title }</div>
+			<div className="wp-block-a8c-github-template__sub-title">{ subtitle }</div>
+			{ body && <div className="wp-block-a8c-github-template__body">{ body }</div> }
+			{ Boolean( labels.length ) && (
+				<ul className="wp-block-a8c-github-template__labels">
+					{ labels.map( ( l ) => (
+						<li key={ l }>{ l }</li>
+					) ) }
+				</ul>
+			) }
+		</El>
+	);
+};
+
+type EditProps = Readonly< {
+	onChangeTitle: ( val: string ) => void;
+	onChangeUserOrOrg: ( val: string ) => void;
+	onChangeRepoName: ( val: string ) => void;
+	onChangeBody: ( val: string ) => void;
+	onChangeLabels: ( val: Label[] ) => void;
+	title: string;
+	userOrOrg: string;
+	repo: string;
+	body: string;
+} >;
+
+const Edit = ( {
+	onChangeTitle,
+	onChangeUserOrOrg,
+	onChangeRepoName,
+	onChangeBody,
+	onChangeLabels,
+	title,
+	userOrOrg,
+	repo,
+	body,
+}: EditProps ): ReactT.ReactElement => {
+	const repoLabels = useMemo< RepoLabels >( () => new Map(), [] );
+	const [ labels, setLabels ] = useState< FormTokenField.Value[] | undefined >( undefined );
+
+	const [ [ bUser, bRepo ] ] = useDebounce( [ userOrOrg, repo ], 250, {
+		equalityFn( l, r ) {
+			return l.every( ( val, i ) => val === r[ i ] );
+		},
+	} );
+
+	useEffect( () => {
+		if ( ! bUser || ! bRepo ) {
+			return;
+		}
+
+		const fullRepo = `${ encodeURIComponent( bUser ) }/${ encodeURIComponent( bRepo ) }`;
+		if ( repoLabels.has( fullRepo ) ) {
+			return;
+		}
+
+		window
+			.fetch( `https://api.github.com/repos/${ fullRepo }/labels?per_page=100`, {
+				headers: {
+					Accept: 'application/vnd.github.v3+json',
+				},
+			} )
+			.then(
+				( resp ) => {
+					if ( resp.status >= 200 && resp.status < 300 ) {
+						resp.json().then( ( data ) => repoLabels.set( fullRepo, data ) );
+						return;
+					}
+					repoLabels.set( fullRepo, null );
+				},
+				() => repoLabels.set( fullRepo, null )
+			);
+	}, [ bUser, bRepo, repoLabels ] );
+
+	return (
+		<>
+			<InspectorControls>
+				<PanelBody title={ __( 'Labels!' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **66** times:
`translate( 'Labels' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/labels-in-github-issues)